lib/internal/bootstrap/node.js

```diff
@@ -297,6 +290,41 @@ function startup() {
     }
   });
 
+  // Set up coverage exit hooks.
+  let originalReallyExit = process.reallyExit;
+  // Core coverage generation using nyc instrumented lib/ files:
+  // see `make coverage-build`.
+  // This does not affect user land.
+  if (global.__coverage__) {
+    const {
+      writeCoverage
+    } = NativeModule.require('internal/coverage-gen/with_instrumentation');
+    process.on('exit', writeCoverage);
+    originalReallyExit = process.reallyExit = (code) => {
+      writeCoverage();
+      originalReallyExit(code);
+    };
+  }
+  // User-facing NODE_V8_COVERAGE environment variable that writes
+  // ScriptCoverage to a specified file.
+  if (process.env.NODE_V8_COVERAGE) {
+    const cwd = NativeModule.require('internal/process/execution').tryGetCwd();
+    const { resolve } = NativeModule.require('path');
+    // Resolve the coverage directory to an absolute path.
+    const coverageDireoctry = resolve(cwd, process.env.NODE_V8_COVERAGE);
```

Comment:
`coverageDireoctry` -> `coverageDirectory`.

Also, worth noting, the reason I was updating `process.env.NODE_V8_COVERAGE` to be the resolved path, was so that when it's passed to a subprocess (which might change `cwd`) coverage continues to write to the same path that was configured during bootstrapping.